content/codepipeline/change.md

```diff
@@ -29,14 +29,17 @@ After you modify and commit your change in GitHub, in approximately one minute y
 
 ![CodePiepline Running](/images/codepipeline/codepipeline_building.png)
 
-Once the service is built and delivered, run the following command to get the Elastic Load Balancer (ELB) endpoint and open it in a browser.
-If the message is not updated immediately, give Kubernetes some time to deploy the change.
+#### Challenge:
+**How can we validate the changes and access latest version of the app?**
+ {{%expand "Expand here to see the solution" %}}
```

Comment:
I would add a HINT
HINT: Which command will get you Elastic Load Balancer (ELB) endpoint for this app?

Reply:
added and pushed